### DISCUSSION TITLE
How to retrieve the annotation for the structures displayed?

### DISCUSSION THREAD

**Discussion by AngelRuizMoreno** [facebookresearch/esm#337] [ESM Metagenomic Atlas]
Is there any way to get the information about the source organism, study or any other annotation for the predicted structures in ESM Metagenomic Atlas?

**Comment by tomsercu** [comment]
Actually there's not much known about those proteins, as the source organism is unknown (hence, "metagenomics", doesn't come from a single organism's genome). The proteins in MGnfy also don't necessarily come from a single study:
> MGnify produces a non-redundant protein database that is generated from amalgamating all open reading frame predictions from the analysis of all of the underlying assembled datasets

From https://academic.oup.com/nar/article/48/D1/D570/5614179?login=false#190997789
